packages/@aws-cdk/aws-events-targets/lib/batch.ts

```diff
@@ -0,0 +1,81 @@
+import * as events from '@aws-cdk/aws-events';
+import * as batch from '@aws-cdk/aws-batch';
+import * as iam from '@aws-cdk/aws-iam';
+import { singletonEventRole } from './util';
+
+/**
+ * Customize the Batch Job Event Target
+ */
+export interface BatchJobProps {
+  /**
+   * The event to send to the Lambda
+   *
+   * This will be the payload sent to the Lambda Function.
+   *
+   * @default the entire CloudWatch event
+   */
+  readonly event?: events.RuleTargetInput;
+
+  /**
+   * The size of the array, if this is an array batch job.
+   * 
+   * Valid values are integers between 2 and 10,000.
+   * 
+   * @default undefined
```

Comment:
The linter should yell at you for this. Doesn't it?

Reply:
fixed